scripts/run

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+# run the service in one of different deployment modes
+set -e #-x
+
+usage() {
+  cat <<-'EOF'
+usage: run [-h|--help] RUNTYPE
+
+[-h|--help]       print this help text
+
+RUNTYPE is one of:
+- tests           run tests against repo
+- integ           run integration tests against running instance;
```

Comment:
Can we go with "integration"? I think it will be hard to remember "integ" given that it's not a standard abbreviation.

Reply:
if these were options, I would adopt the `--t` and `--i` in a heartbeat, but this is a positional argument for `RUNTYPE`. moved to `integration`